.ci/docker/common/install_mkl.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
@atalman Any plans to sync this up with the mkl version being installed via [install_conda.sh](https://github.com/pytorch/pytorch/blob/46659a388ce9791575d598b8cfc68b5b8cfa7c1f/.ci/docker/common/install_conda.sh#L86). There is a version difference (2021.4.0 for `install_conda.sh` vs 2024.2.0 for this file), a dynamic (`install_conda.sh`) vs static library (this file) difference, as well as an install location difference (`/opt/conda/envs/py_$ANACONDA_PYTHON_VERSION` for `install_conda.sh` vs `/opt/intel` for this file) .

Reply:
hi @jithunnair-amd yes totally we should try to move to 2024.2.0. conda install will not work since we are not using conda-forge. However we should install these packages from pip